js/constants.js

```diff
@@ -48,13 +48,19 @@ window.constants = (function () {
   ];
 
   var WIZARD_SIMILAR_COUNT = 4;
+  var DATA_REQUEST_GET_URL = 'https://javascript.pages.academy/code-and-magick/data';
+  var DATA_REQUEST_POST_URL = 'https://javascript.pages.academy/code-and-magick';
```

Comment:
а есть у нас будет 10 разных запросов, будешь для каждого прописывать полный URL ?

Reply:
```suggestion
  var API_URL = 'https://javascript.pages.academy/code-and-magick';
```
А в методе используем `API_URL + '/data'`